Docs/source/Software/Tasks.rst

```diff
@@ -2,6 +2,35 @@
 Real Time Operating System Tasks
 ********************************
 
+Initialization Task: Alex Koo
+===============================
+
+Purpose
+    Create tasks by calling "OSTaskCreate()" and provide argument specifying to RTOS how the task will be managed.
+    The order of the semaphore Initialization matters because if the fault semaphore isn't initialized first and a fault is called we do not go into the fault state.
+    Initialization Task also creates :term: 'Semaphores <Semaphore>' which are used when a task wants exclusive 
```

Comment:
There shouldn't be a space in between : `. Also you are using apostrophes and you should be using `. It is the key on your keyboard that also has ~. After running make HTML, go into the build directory in docs and click on any web page. Then you can go to what you changed and see if it worked the way you intended.